impl/src/main/java/org/jboss/weld/bootstrap/events/ProcessSimpleInjectionTarget.java

```diff
@@ -45,6 +45,7 @@ public ProcessSimpleInjectionTarget(BeanManagerImpl beanManager, AnnotatedType<X
     public void setInjectionTarget(InjectionTarget<X> injectionTarget) {
         checkWithinObserverNotification();
         this.injectionTarget = injectionTarget;
+        BootstrapLogger.LOG.setInjectionTargetCalled(getReceiver().getClass(), injectionTarget.getClass());
```

Comment:
I think `injectionTarget.getClass()` does not make sense here. It will be better to simply pass the IT instance. And the same probably applies to extension. Also I think it would be better to log both the original and the new value for all `ProcessX` events, e.g. something like:
`ProcessInjectionTarget.setInjectionTarget() called by {0}: {1} replaced by {2}`. What do you think?